philly/spider-0428-base.yaml

```diff
@@ -0,0 +1,25 @@
+description: Spider baseline (v0428, stability)
+
+auth:
+  vc: msrlabs
+  cluster: wu2
+  docker:
+    image: pytorch/pytorch:0.4.1-cuda9-cudnn7-devel
+
+storage:
+  _default:
+    storage_account_name: polozovphillyblob
```

Comment:
This sort of configuration probably shouldn't be committed (or placed in a separate path indicating it's just for you).